doc/en/example/nonpython/conftest.py

```diff
@@ -2,17 +2,17 @@
 import pytest
 
 
-def pytest_collect_file(parent, path):
-    if path.ext == ".yaml" and path.basename.startswith("test"):
-        return YamlFile.from_parent(parent, fspath=path)
+def pytest_collect_file(parent, fspath):
```

Comment:
seeing this reminds me - adding those parameters is a breaking changes for externals that use those,

i believe it adds errors on some plugins, please see #8361